uber/utils.py

```diff
@@ -477,6 +477,31 @@ def get_real_badge_type(badge_type):
     return c.ATTENDEE_BADGE if badge_type in [c.PSEUDO_DEALER_BADGE, c.PSEUDO_GROUP_BADGE] else badge_type
 
 
+def add_opt(opts, other):
```

Comment:
This is kind of the meat of the change. To try to explain it succinctly:

While we use a custom type for the column itself, the value we actually work with when we manipulate something like `attendee.ribbon` is just a comma-separated string. In order for this refactor to work, we needed to easily do three things:
1. Be able to see if `attendee.ribbon` contained a value, a problem we solved in config.py by converting all the ints to strings.
2. Be able to add a value to attendee.ribbon, even if it's already there (the latter problem is solved by that set() conversion I mentioned earlier).
3. Be able to remove a value from attendee.ribbon, even if it's not there.

Unfortunately for us, you can't actually add new operands to built-in classes like strings, so rather than being able to say `attendee.ribbon.add_opt(blah)` we have to just use plain ol' functions. It's less pretty, but it works.